docs/framework/react/guides/next-js.md

```diff
@@ -0,0 +1,187 @@
+---
+id: next-js
+title: Next JS Usage
+---
+
+# Using TanStack Form in a Next.js App Router
+
+TanStack Form is compatible with React out of the box, supporting `SSR` and being framework-agnostic. However, specific configurations are necessary according to your chosen framework.
```

Comment:
Let's add a link before the start of the content linking out to this blog series I wrote for the topic:

https://unicorn-utterances.com/collections/react-beyond-the-render

> Before reading this guide, it's suggested you understand how React Server Components and React Server Actions work. [Check out this blog series for more information.](https://unicorn-utterances.com/collections/react-beyond-the-render)